### PR TITLE
feat(regulations): Regulations/UI addons

### PR DESCRIPTION
**Pull request by Valur**
## What

Add ui tweaks used in regulations admin and update some related code

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by thordurhhh — APPROVED** [review]
👍

File: libs/island-ui/core/src/lib/Tag/Tag.tsx [hunk 1]
```diff
@@ -79,10 +87,12 @@ export const Tag = forwardRef<HTMLButtonElement & HTMLAnchorElement, TagProps>(
     const hyphenated = hyphenate && typeof children === 'string' && (
       <Hyphen>{children}</Hyphen>
     )
+    const renderCloseIcon = removable && !!onClick && !href
 
     const content = (
       <Text variant="eyebrow" as="span" truncate={truncate}>
         {!truncate && hyphenate ? hyphenated : children}
+        {renderCloseIcon && <Icon icon="close" className={styles.closeIcon} />}
```
Comment (Arnarkari93) [inline libs/island-ui/core/src/lib/Tag/Tag.tsx:95]: I think this should not be implemented in `island-ui` until the designs in Figma have been updated. If this should be added we need to account for accessibility.

**Review by Arnarkari93 — CHANGES_REQUESTED** [review]

File: libs/island-ui/core/src/lib/Tag/Tag.tsx (hunk 1 above)
Comment (Arnarkari93) [reply libs/island-ui/core/src/lib/Tag/Tag.tsx:95]: I think we should instead add a close icon by allowing the user to inject a custom icon instead—that way we allow for more flexibility for different use-cases.

**Review by saevarma — COMMENTED** [review]
Why does this PR include the `package.json` changes?

**Pull request closed without merging**